File: SSServer/src/main/java/com/prose/crhen/SSServer/db/ServerHistoryRepository.java
package com.prose.crhen.SSServer.db;

import com.prose.crhen.SSServer.model.Server;
import com.prose.crhen.SSServer.model.ServerHistory;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface ServerHistoryRepository extends JpaRepository<ServerHistory, Long> {

    public List<ServerHistory> findByServer(Server server);
}

File: SSKotlinServer/src/main/kotlin/com/prose/crhen/SSKotlinServer/SSKotlinServer/db/ServerHistoryRepository.kt
package com.prose.crhen.SSKotlinServer.SSKotlinServer.db

import com.prose.crhen.SSKotlinServer.SSKotlinServer.model.Server
import com.prose.crhen.SSKotlinServer.SSKotlinServer.model.ServerHistory
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ServerHistoryRepository: JpaRepository<ServerHistory, Long> {
    fun findByServer(server: Server): List<ServerHistory>
}
